pkg/cloudprovider/cloud.go

```diff
@@ -33,6 +33,8 @@ type Interface interface {
 	Zones() (Zones, bool)
 	// Clusters returns a clusters interface.  Also returns true if the interface is supported, false otherwise.
 	Clusters() (Clusters, bool)
+	// GetProviderID returns the cloud provider ID.
+	GetProviderID() string
```

Comment:
"Getter" functions in go don't start with "Get" so you should rename this to "ProviderID"